internal/common/node_module_info.bzl

```diff
@@ -38,6 +38,8 @@ NodeModuleInfo = provider(
 # NodeModuleSources provider is provided by targets that are npm dependencies by the
 # `node_module_library` rule as well as other targets that have direct or transitive deps on
 # `node_module_library` targets via the `collect_node_modules_aspect` below.
+# TODO: rename to NodeModuleSourcesInfo so name doesn't trigger name-conventions warning
```

Comment:
@alexeagle It looks like the provider names ending with `Info` is the convention or it trips up the `name-conventions` warnings although I couldn't find that anywhere in the Bazel docs at first glance except for under the `rule` docs:

```
provides | ... Each element of the list is an `*Info` object returned by provider(), except that a legacy provider is represented by its string name instead.
```

Reply:
sure, but let's not spend too much effort satisfying a linter, the benefits are so small

Reply:
Ok. Will rename in a separate PR. It's only used in a few places and it's better to stick to naming conventions.